src/versions.ts

```diff
@@ -57,19 +59,22 @@ export interface Versions {
   /** Full list of all known Electron releases, Sorted in branch order. */
   readonly versions: SemVer[];
 
-  /** @returns true iff `version` is a release that this object knows about */
+  /** Returns true iff `version` is a release that this object knows about */
   isVersion(version: SemOrStr): boolean;
 
-  /** @returns all versions matching that major number. Sorted in branch order. */
+  /** Returns all versions matching that major number. Sorted in branch order. */
   inMajor(major: number): SemVer[];
 
-  /** @returns all versions in a range, inclusive. Sorted in branch order. */
+  /** Return all versions in a range, inclusive. Sorted in branch order. */
   inRange(a: SemOrStr, b: SemOrStr): SemVer[];
 
-  /** @returns {@link ReleaseInfo} iff `version` is a release that this object knows about */
+  /** Retrieves the release info */
   getReleaseInfo(version: SemOrStr): ReleaseInfo | undefined;
 }
```

Comment:
It looks like these are marked as undocumented due to https://github.com/microsoft/rushstack/issues/3462 - I'd prefer to leave them as they were, the upstream bug will (hopefully) be fixed one day.

Reply:
Fair enough. Done 👍